packages/flutter/lib/src/widgets/safe_area.dart

```diff
@@ -25,6 +25,23 @@ import 'media_query.dart';
 ///
 /// {@youtube 560 315 https://www.youtube.com/watch?v=lkF0TQJO0bA}
 ///
+/// {@tool snippet}
+///
+/// This example creates a blue box containing text that is sufficiently padded
+/// to avoid instrusions by the operating system.
+///
+/// ```dart
+/// SafeArea(
+///   child: Container(
+///     constraints: const BoxConstraints.expand(),
+///     alignment: Alignment.center,
+///     color: Colors.blue,
+///     child: const Text('Hello, World!'),
+///   ),
+/// )
+/// ```
+/// {@end-tool}
+///
 /// See also:
```

Comment:
[`MediaQueryData.padding`](https://main-api.flutter.dev/flutter/widgets/MediaQueryData/padding.html) has a really informative video about this topic—maybe it'd be nice to link there in these docs.

Reply:
Excellent suggestion, added this video these docs too!